### ISSUE TITLE
Karma 0.13.20 does not detect files when using webpack

### ISSUE THREAD

**Issue by ianks** [karma-runner/karma#1894]
After upgrading to 0.13.20, none of my tests ran. Sorry I cannot provide too much debugging information, but I don't know where to start.

Here is my karma.conf.js

```js
var webpackConfig = require('./../config/webpack.config')
var path = require('path')

module.exports = function (config) {
  config.set({
    basePath: path.join(__dirname, '..'),
    autoWatch: true,
    logLevel: config.LOG_INFO,
    browsers: [ 'PhantomJS' ],
    files: [
      'http://cdn.auth0.com/js/lock-8.2.1.min.js',
      'test/index.js'
    ],
    port: 8080,
    captureTimeout: 60000,
    frameworks: [ 'phantomjs-shim', 'mocha', 'chai', 'sinon-chai' ],
    client: {
      mocha: {}
    },
    reporters: [ 'mocha' ],
    preprocessors: {
      'test/index.js': [ 'webpack', 'sourcemap' ]
    },
    webpack: webpackConfig,
    webpackServer: {
      noInfo: true
    }
  })
}
```

With a simple index.js for webpack

```js
require('es6-promise').polyfill();
const testsContext = require.context('../.tmp/test', true, /Test$/);
testsContext.keys().forEach(testsContext);
```

**Comment by filipearena** [comment]
+1

**Comment by americoneto1** [comment]
+1

**Comment by dignifiedquire** [comment]
This is due to https://github.com/karma-runner/karma/pull/1791 @budde377 could you check this out please if you get a moment?

**Comment by dignifiedquire** [comment]
Released `0.13.21` that reverts the problematic commit.

**Comment by budde377** [comment]
Will do!
